packages/11ty/_includes/README.md

```diff
@@ -1,6 +1,51 @@
-## Eleventy layout templates
+## Eleventy `_includes`
 
-> Eleventy Layouts are special templates that can be used to wrap other content. To denote that a piece of content should be wrapped in a template, use the layout key in your front matter.
+> Eleventy `_includes`, like `_layouts` are language-agnostic, are not processed as full template files, and are meant to wrap and/or be wrapped by other templates. Unlike `_layouts`, `_includes` are meant to be consumed _inline_ by other templates such as `_layouts` or other `_includes`.
 
-See also the [Eleventy layouts document](https://www.11ty.dev/docs/layouts/)
+### Data
+All `_includes` have access to two parameters, `eleventyConfig` and `globalData`, which will be followed by any `args` passed to the template when it's being included.
 
+#### eleventyConfig
+The `eleventyConfig` object
+
+#### globalData
+Data from the yaml files in `content/_data`
+
+### Example Usage
+
+#### In an `11ty.js` `_include`
+```
+module.exports = function(eleventyConfig, globalData, page) {
+  const siteTitle = eleventyConfig.getFilter('siteTitle')
+
+  return `
```

Comment:
This should also use the `common-tags` methods to render the string template literal.